Controllers/userController.js

```diff
@@ -0,0 +1,67 @@
+const User = require('../Models/userModel');
+
+const getAllUsers = (req, res) =>{
+    User.find({}).then(user => {
+        res.status(200).json({
+            status: "ok",
+            user
+        })
+    })
+};
+
```

Comment:
Add getUserAppointmentsByID function
We need to be able to pull all of the appointments that are relevant to a specific user.  something like:
```suggestion

const getAuthorsWithCookbooks = (req, res) => {
     Author.find({ _id: req.params.id })
     .populate("cookbooks")
     .then((authors) => {
         res.json(authors);
     });
};


```